analysis/06_additional_trigger_analysis.md

```diff
@@ -0,0 +1,991 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.5
+  kernelspec:
+    display_name: bgdenv
+    language: python
+    name: python3
+---
+
+# Additional Analysis Requested by Hassan
+
+
+This notebook covers 3 analyses requested by Hassan for the trigger.
+
+- The FAR and POD calculated at Bahadurabad, Hardinge Bridge, and Goalando, in Excel file form. Note that for Goalando he wants us to compute it by (1) a point on the GloFAS raster, (2) by combining the discharge at Hardinge bridge with 1 day lag time with that at Bahadurabad with 2 day lag time
+- Calculate the probability of flooding at Bahadurabad if there is flooding at Goalando, and vice versa
+- Calculate 1-in-5 and 1-in-10 year RPs at Goalando for both GloFAS and FFWC data
+
+```python
+%load_ext jupyter_black
+```
+
+```python
+from pathlib import Path
+import os
+
+import matplotlib.backends.backend_pdf
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+os.chdir("..")
```

Comment:
This shouldn't be needed if you run `pip install -e .` in the top-level directory